Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -47,7 +51,15 @@ class DisplayHymnViewModel: ObservableObject {
             }).store(in: &disposables)
     }
 
+    func updateFavorite() {
+        if self.favoritedStatus {
+            FavoritedHymn.saveFavorite(hymnType: self.hymnType, hymnNumber: self.hymnNumber)
+        } else {
+            FavoritedHymn.removeFavorite(hymnType: self.hymnType, hymnNumber: self.hymnNumber)
+        }
+    }
+
     func toggleFavorited() {
-        self.favorited.toggle()
+        self.favoritedStatus.toggle()
```

Comment:
i think it would make sense to combine these two methods. toggling can just save/unsave

Reply:
Resolved in commit 317ca8d